MapboxDirections/MBRouteOptions.swift

```diff
@@ -304,6 +350,22 @@ open class RouteOptions: NSObject, NSSecureCoding {
     open var includesExitRoundaboutManeuver = false
     
     /**
+     A Boolean value indicating whether each route step includes an array of `SpokenInstructions`.
+     
+     If this option is set to true, the `RouteStep.instructionsSpokenAlongStep` property is set to an array of `SpokenInstructions`.
+     */
+    open var includesVoiceInstructions = false
+    
+    /**
+     The measurement system used in spoken instructions included in route steps.
+     
+     If the `includesSpokenInstructions` property is set to `true`, this property determines the units used for measuring the distance remaining until an upcoming maneuver. If the `includesSpokenInstructions` property is set to `false`, this property has no effect.
+     
+     You should choose a measurement system appropriate for the current region. You can also allow the user to indicate their preferred measurement system via a setting.
+     */
+    open var distanceMeasurementSystem: MeasurementSystem = .metric
```

Comment:
Flagging here that I think https://github.com/mapbox/MapboxDirections.swift/pull/175#discussion_r139206621 still applies, that we should do something based on #133 instead of making the developer figure out the right measurement system.